Source/SwiftLintBuiltInRules/Rules/Performance/EmptyCountRule.swift

```diff
@@ -62,6 +81,40 @@ struct EmptyCountRule: Rule {
                 Example("isEmpty && [Int]().isEmpty"),
             Example("[Int]().count != 3 && [Int]().↓count != 0 || ↓count == 0 && [Int]().count > 2"):
                 Example("[Int]().count != 3 && ![Int]().isEmpty || isEmpty && [Int]().count > 2"),
+            Example("#ExampleMacro { $0.list.↓count == 0 }"):
+                Example("#ExampleMacro { $0.list.isEmpty }"),
+            Example("#Rule { $0.donations.↓count == 0 }", excludeFromDocumentation: true):
+                Example("#Rule { $0.donations.isEmpty }"),
+            Example(
+                "#Rule(param1: \"param1\", param2: \"param2\") { $0.donations.↓count == 0 }",
+                excludeFromDocumentation: true
+            ): Example(
+                    "#Rule(param1: \"param1\", param2: \"param2\") { $0.donations.isEmpty }"
+                ),
+            Example(
+                "#Rule(param1: \"param1\") { $0.donations.↓count == 0 } closure2: { doSomething() }",
+                excludeFromDocumentation: true
+            ): Example(
+                "#Rule(param1: \"param1\") { $0.donations.isEmpty } closure2: { doSomething() }"
+                ),
+            Example(
+                "#Rule(param1: \"param1\") { return $0.donations.↓count == 0 }",
+                excludeFromDocumentation: true
+            ): Example(
+                "#Rule(param1: \"param1\") { return $0.donations.isEmpty }"
+                ),
+            Example("""
+                #Rule(param1: "param1") {
+                    doSomething()
+                    return $0.donations.↓count == 0
+                }
+            """, excludeFromDocumentation: true):
+                Example("""
+                #Rule(param1: "param1") {
+                    doSomething()
+                    return $0.donations.isEmpty
+                }
+            """),
```

Comment:
To verify the correction mechanism, one test case is enough. We don't need to repeat all of them here again.